ts/parts-more/Pane.ts

```diff
@@ -313,6 +313,19 @@ extend(Pane.prototype, {
          */
         size: '85%',
 
+        /**
+         * The size of pane's inner area (area in the center that is not
+         * occupied), either as a number defining pixels, or a
+         * percentage defining a percentage of the plot area.
```

Comment:
If I remember correctly, it is a percentage of the pane's _radius_, as defined in the `size` option. This should be communicated more precisely.

Reply:
I thought that it is a similar way as `size`. The `size` descriptions says: _'The size of the pane, either as a number defining pixels, or a percentage defining a percentage of the plot are.'_ but I will change that. Thanks!

Reply:
Yes, actually the size is a percentage of the smallest of the plot height or plot width. So a `size` og 100% fills the smallest dimension of the plot area.

Reply:
I have corrected the description for both, `size` and `innerSize`. In case when the `innerSize` is percentage, it is percentage of the `size`.